app/components/LoadMore.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { useInView } from "react-intersection-observer";
+import Spinner from './Spinner';
+import Post from "./Post";
+import fetchPosts from "../utils/fetchPosts"
+
+/*TODO: Instead of passing in postData as a Prop to this component, this component
+ * should call an API route that returns the posts.
+ */
```

Comment:
```suggestion
/* TODO: Instead of passing in postData as a Prop to this component, this component
 * should call an API route that returns the posts.
 */
```